README.md

```diff
@@ -1,2 +1,83 @@
+
+## My philosophy
+
+I often ask people "What is your purpose in life?" I think it's a very telling question and I hope that you will one day be able to answer what you want to do when you grow up! For me, the answer has been pretty consistant for the past few years: **My purpose in life is to help people become the best they can be.**
+
+I have always had a strong drive to help people and do the right thing. This often gets me into trouble more often than I like to admit (if it's not a HELL YES, it's a no), but I rarely regret it. I have a very strong moral compass, which can make it either easy or difficult to work with me! Doing the right thing is often hard, but it's the right thing.
+
+[Radical Candor](https://www.blinkist.com/en/books/radical-candor-en/) is perhaps the best resource to understand my approach to management and feedback. I believe in giving timely, honest feedback _because I care about you._ I don't believe in surprising people at review time with bad feedback. Your performance reviews should be underwhelming events because you should always know where you stand with me at all times. You can read more about my thoughts on feedback [here](https://medium.com/@tsunamino/setting-goals-with-your-engineers-that-dont-completely-suck-cb76b87e4275).
+
+## My role
+
+I like to think of myself as a momma bear coaching a soccer team that my cubs are on. I will 100% eat any other bears that dares to hurt my team, but I also expect my cubsters to be on their best behavior and to be excellent team players. Everyone must shake hands with the other team at the end of the game. No exceptions.
+
+In real people terms, it means I expect my reports to be collaborative team players and to be good sports if something does go wrong. There is no room for ego on my team. **There is no room for bullies on this team.** If someone is doing really well on the team, we should all celebrate that person's work! If you're doing really well, we will also celebrate you! If you need to constantly be in the limelight **at the expense of others**, then you may want to reconsider if this is the best fit for you.
+
+My role as the EM on this team is to:
+
+- unblock you
+- provide you with a safe environment to grow into a well-rounded human
+- give you timely feedback, both good and constructive
+- get you the resources you need
+- challenge you in ways that will help you grow
+- champion your accomplishments to anyone who will listen (and even those who won't)
+
+My role is not to:
+
+- do your job for you
+- enable laziness
+
+You do have to meet me halfway. I will be your biggest cheerleader, but I can't be everywhere at once! Sometimes I'll ask you to write down what your doing to help me out. Sometimes it's because I legit have too many things to focus on and I need an easy way to recall all the amazing things that you've done lately. Help me help you get more money!
```

Comment:
"what your doing" => "what you're doing"

Reply:
Yikes. I must've been real tired